src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/AuthenticationController.kt

```diff
@@ -26,8 +27,10 @@ class AuthenticationController(context: ControllerContext, htmlFormHelpers: Form
             oneRepoEndpoint("authenticate/", this::authenticate, repos.user, HttpMethod.post)
                     .withAdditionalSetup({ url, _ -> setupSecurity(url) })
```

Comment:
I'm now seeing a compiler error on this line! passed function doesn't match expected signature